web/src/main/java/com/graphhopper/http/GraphHopperServlet.java

```diff
@@ -112,6 +113,8 @@ public void doGet(HttpServletRequest httpReq, HttpServletResponse httpRes) throw
                     throw new IllegalArgumentException("If you pass " + POINT_HINT + ", you need to pass a hint for every point, empty hints will be ignored");
                 }
 
+                List<String> pathDetails = new ArrayList<String>(Arrays.asList(getParams(httpReq, PATH_DETAILS)));
```

Comment:
minor: the wrapper `new ArrayList` should not be necessary. If IntelliJ warns you just do `Arrays.<String>asList`?